core-bundle/contao/library/Contao/Image.php

```diff
@@ -113,13 +98,7 @@ public static function getPath($src)
 			trigger_deprecation('contao/core-bundle', '5.2', 'Using the "%s" icon has been deprecated and will no longer work in Contao 6. Use the "%s--disabled" icon instead.', $filename, substr($filename, 0, -1));
 		}
 
-		// Prefer SVG icons
-		if (file_exists($projectDir . '/system/themes/' . $theme . '/icons/' . $filename . '.svg'))
-		{
-			return 'system/themes/' . $theme . '/icons/' . $filename . '.svg';
-		}
```

Comment:
Seems to be unrelated to this PR and breaks BC.

Reply:
See https://github.com/contao/contao/pull/7094#discussion_r1611746056

Reply:
I agree that we have to keep this BC layer for old theme icons.

Reply:
Restored in b9e5f051394692fedb1a90cf8e9dded6066e3d85.